core/ir/aarch64/codec.c

```diff
@@ -6824,13 +6824,6 @@ encode_svemem_vec_imm5(uint enc, aarch64_reg_offset element_size, bool is_prefet
         get_vector_element_reg_offset(opnd) != element_size)
         return false;
 
-    bool index_scaled;
-    uint index_scale_amount;
-    if (opnd_get_index_extend(opnd, &index_scaled, &index_scale_amount) !=
-            DR_EXTEND_UXTX ||
-        index_scaled || index_scale_amount != 0)
-        return false;
-
```

Comment:
Again, removing code from the AArch64 decoder looks like a mistake?

Reply:
Ditto below.